tests/conformance_ruby/tags/include_tag_test.rs

```diff
@@ -81,33 +82,41 @@ fn test_include_tag_for() {
 }
 
 #[test]
-#[should_panic] // liquid-rust#237
+fn test_include_tag_with_single_local_variable() {
+    assert_template_result!(
+        "Single Locale: test123",
+        "{% include 'single_locale_variable' echo1:'test123' %}",
+        o!({}),
+        liquid()
+    );
+}
+
+#[test]
+#[should_panic]
 fn test_include_tag_with_local_variables() {
     assert_template_result!(
-        "Locale: test123 ",
-        "{% include 'locale_variables' echo1: 'test123' %}",
+        "Locale: test123",
+        "{% include 'locale_variables' echo1:'test123' %}",
         o!({}),
         liquid()
     );
 }
 
 #[test]
-#[should_panic] // liquid-rust#237
 fn test_include_tag_with_multiple_local_variables() {
     assert_template_result!(
         "Locale: test123 test321",
-        "{% include 'locale_variables' echo1: 'test123', echo2: 'test321' %}",
+        "{% include 'locale_variables' echo1:'test123' echo2:'test321' %}",
```

Comment:
It looks like there are still some syntax gaps we need to handle.

Feel free to split this PR into two and we can get the Jekyll half landed immediately.